demo/chat/package.json

```diff
@@ -3,14 +3,15 @@
   "version": "1.0.1",
   "description": "Demo app for for Amazon Chime Chat SDK",
   "scripts": {
-    "create": "webpack",
-    "start": "npm install && webpack-dev-server",
-    "test": "test"
+    "deps": "cd ../.. && npm run build && npm pack",
+    "start": "npm run deps && npm install && webpack-dev-server",
+    "start:fast": "webpack-dev-server"
   },
   "dependencies": {
     "@aws-amplify/auth": "^3.4.17",
     "@aws-amplify/core": "^3.8.9",
     "@aws-amplify/storage": "^3.3.17",
+    "amazon-chime-sdk-component-library-react": "file:../../amazon-chime-sdk-component-library-react-1.6.0.tgz",
```

Comment:
Is there a reason to reference the `.tgz` instead of the relative path to the module? Like we started doing with [the SDK repo demo](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/package.json#L34)

Reply:
When I tested it out, if we just pointed to the parent folder, it copies everything (parent's node_modules, demo files, etc.) so thats why I added npm pack to only copy over the lib folder.

Reply:
That means we'll have to npm pack this module to use it?

Reply:
yes that is why I added npm run deps to go to the parent folder and packed it.

Reply:
Does a customer need to know about that? I guess I'm wondering what the difference between this and a customer installing `amazon-chime-sdk-component-library-react` directly through npm. Are customers getting needless extra files?

Reply:
Ideally we want an easy path for devs to depend on relative path of the components lib better dev experience and have live reloading. One way is to setup different web-pack configs one for release and one for development.

Reply:
> Does a customer need to know about that? I guess I'm wondering what the difference between this and a customer installing `amazon-chime-sdk-component-library-react` directly through npm. Are customers getting needless extra files?

@p-foucht  When a customer runs `npm install amazon-chime-sdk-component-library-react`, `node_modules/amazon-chime-sdk-component-library-react` includes only files and folders listed in the `files` field of `package.json` plus some mandatory files like `README.md`.
https://github.com/aws/amazon-chime-sdk-component-library-react/blob/aff5c53a266620825dfd5611ec6c4c272cbc96e2/package.json#L7-L9

Based on what Trung said, the demo's `node_modules/amazon-chime-sdk-component-library-react` includes all folders and files.

@ltrung Am I understanding correctly? Have you checked the NPM documentation about this behavior?

Reply:
@simmkyu Yes, it is documented here https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files. From what I understand, it just copied everything that is not specified in .npmignore

Reply:
> @simmkyu Yes, it is documented here https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files. From what I understand, it just copied everything that is not specified in .npmignore

In the `package.json#files` doc, I can't find any description that says installing a local NPM folder as a dependency includes all files. I think `package.json#files` has the top priority, so, ideally, it should only include files and folders listed in `package.json#files`.

Reply:
I just tried running `npm install` after specifying `file:../..`. In `node_modules`, it creates an alias to the folder, so you see all files and folders. What was the issue with `file:../..` again?

Reply:
Switch to using yalc to simulate installing a publish package so it is closer to how a real package will use the library